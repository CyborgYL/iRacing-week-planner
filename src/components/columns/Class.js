// @flow

import * as React from 'react';
import LicenceLevel from '../LicenceLevel';

type Props = {
  race: {
    licenceLevel: number,
  }
};

export default function Class({ race }: Props) {
  return (
    <td><LicenceLevel effective licence={race.licenceLevel} /></td>
  );
}
