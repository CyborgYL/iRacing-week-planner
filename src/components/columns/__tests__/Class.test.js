import React from 'react';
import { shallow } from 'enzyme';
import { describe, test } from '@jest/globals';

import Class from '../Class';

describe('components/columns/Class', () => {
  test('renders correctly', () => {
    const component = shallow(<Class race={{ licenceLevel: 5 }} />);

    expect(component).toMatchSnapshot();
  });
});
