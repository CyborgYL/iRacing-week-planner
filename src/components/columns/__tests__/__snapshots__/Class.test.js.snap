// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/columns/Class renders correctly 1`] = `
<td>
  <LicenceLevel
    effective={true}
    licence={5}
  />
</td>
`;
